src/transports/http.c

```diff
@@ -620,7 +620,7 @@ static int http_connect(http_subtransport *t)
 
 	error = git_stream_connect(t->io);
 
-#if defined(GIT_OPENSSL) || defined(GIT_SECURE_TRANSPORT) || defined(GIT_CURL)
+#if defined(GIT_OPENSSL) || defined(GIT_SECURE_TRANSPORT) || defined(GIT_CURL) || defined(GIT_CUSTOM_TLS)
```

Comment:
Since the stream is pluggable, it doesn't make sense to have this line either. Either we have a stream and need to run this code or it'll error out before we get here.